lib/src/DateRangePicker/DateRangePicker.tsx

```diff
@@ -56,79 +56,70 @@ export function makeRangePicker<TWrapper extends SomeWrapper>(Wrapper: TWrapper)
 
   function RangePickerWithStateAndWrapper({
     calendars,
-    minDate,
-    maxDate,
-    disablePast,
-    disableFuture,
-    shouldDisableDate,
-    showDaysOutsideCurrentMonth,
-    onMonthChange,
-    disableHighlightToday,
-    reduceAnimations,
     value,
     onChange,
     mask = '__/__/____',
     startText = 'Start',
     endText = 'End',
     inputFormat: passedInputFormat,
-    ...restPropsForTextField
+    minDate: __minDate = defaultMinDate,
```

Comment:
For consistency

```suggestion
    minDate: minDateProp = defaultMinDate,
```

Reply:
Doesn't it look less clear? Because accidentally type `minDateProp` when you are typing `minDate` is much much easier than `__minDate`. And `__` is some kind of convention about naming private variables?

Reply:
No specific preference, we could make it the new convention, do you want to update all the other components?

Reply:
Probably yes :) 
I`ll give you more context: Here we had exactly the same naming `minDateProp` but it was problematic because appeared in the autocomplete for `minDate` so I decided to use another convention